Backend/lib/yudhishthira/src/Lib/Yudhishthira/Types.hs

```diff
@@ -144,6 +148,7 @@ instance Show LogicDomain where
   show POOLING = "POOLING"
   show FARE_POLICY = "FARE-POLICY"
   show DYNAMIC_PRICING_UNIFIED = "DYNAMIC_PRICING_UNIFIED"
+  show (CONFIG configType) = "CONFIG" ++ show configType
```

Comment:
`show configType` will make ("\"DRIVER_POOL_CONFIG\"') and please add a separator `_`